utils/chain-ops/genesis/testdata/test-deploy-config-devnet-l1.json

```diff
@@ -38,9 +38,9 @@
   "colosseumSegmentsLengths": "3,11",
   "securityCouncilNumConfirmationRequired": 1,
```

Comment:
We don't need this one. Please remove all variables named `SecurityCouncilNumConfirmationRequired` and `securityCouncilNumConfirmationRequired`